locale/en-us/dialog/ask_enable.dialog

```diff
@@ -0,0 +1 @@
+Just to confirm, you want me to change who I am? I will answer to {ww} and have {descriptor}
```

Comment:
Missing closing punctuation. This causes issues with some TTS (Mozilla iirc repeats the last phoneme if there's no punctuation)